armory/datasets/build.py

```diff
@@ -127,11 +127,16 @@ def build_info(name: str, version: str = None, data_dir: str = None):
     """
     if data_dir is None:
         data_dir = common.get_root()
-    builder = tfds.builder(name, version=version, data_dir=data_dir)
+    # including version as an argument will return True for tfds.core.load._try_load_from_files_first,
```

Comment:
this comment was helpful for me reviewing, but may add more confusion than clarity going forward, since it's referencing an argument that will no longer be included. I would delete

Reply:
resolved with 23c35782